Bugfix/mainpage background

BlackMorzan opened the pull request:
Changed padding in style.css so in max resolution picture should fit in visible area.

## Description
Picture was upscaled in y axis to meet size of vertical padding of 395px, text and counter. That also make picture bigger in x axis and enter invisible space on the right. I guess verticall padding anticipated smaller couter, or none at all.

## Motivation and context
It should make Santa visible - fix to closes #388 

## How has this been tested?
I just eyeballed it for now.

## Screenshots (if appropriate)
![Bez tytułu](https://user-images.githubusercontent.com/44707515/94573412-5aede700-0272-11eb-9842-8de469db67fe.png)
![FB_Button](https://user-images.githubusercontent.com/44707515/94724497-ac71a100-035a-11eb-9579-9441f5a1a573.png)
![Bez tytułu](https://user-images.githubusercontent.com/44707515/94724578-ca3f0600-035a-11eb-8386-709d8f38fd77.png)
![Santa 1024](https://user-images.githubusercontent.com/44707515/95196858-fc2bee80-07d8-11eb-944d-d3c95d05224f.png)
![Santa 1336](https://user-images.githubusercontent.com/44707515/95196870-00580c00-07d9-11eb-8e11-d44479dcec31.png)
![Santa 1440](https://user-images.githubusercontent.com/44707515/95196874-0221cf80-07d9-11eb-9ac6-3262c7c44448.png)



## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

WTobor commented:
- why have you deleted AdminServiceTest.cs?
- is that solution with size change working on the phone and PC?

BlackMorzan commented on src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:
```diff
@@ -2064,8 +2064,8 @@ footer{
 	background-size: cover;
 	background-repeat: no-repeat;
 	overflow: hidden;
```
This chunk of code is related only to devices with screen size over 992px. For resolution between 768px and 992px padding changes again and for lower res site is showed in "mobile device mode" where Santa is is far beyond screen. For more dynamic site we might need someone more skilled in css magic :)

BlackMorzan commented on src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css:
```diff
@@ -2064,8 +2064,8 @@ footer{
 	background-repeat: no-repeat;
 	overflow: hidden;
-	padding-top: 205px;
-	padding-bottom: 190px;
+	padding-top: 130px;
+	padding-bottom: 140px;
```
This chunk of code is related only to devices with screen size over 992px. For resolution between 768px and 992px padding changes again and for lower res site is showed in "mobile device mode" where Santa is is far beyond screen. For more dynamic site we might need someone more skilled in css magic :)

BlackMorzan replied:
I am unsure how this site was intended to react to changing resolution therefore I don't know the right approach to take while debugging.

WTobor replied:
you can show screenshots how does it look like on different resolution in a browser (F12 and Toggle device toolbar)

BlackMorzan replied:
Added screenshots of "mobile device mode" in screenshots section

BlackMorzan replied:
Ok. Different approach is needed

WTobor replied:
screenshots look ok - maybe not perfect, but ok ;)

BlackMorzan replied:
I wasn't happy how it looked on devices over 1200px wide. It required a bit of learning of css but now it looks ok :)

BlackMorzan commented:
> * is that solution with size change working on the phone and PC?
Deletion of AdminServiceTest.cs seems related to merge with develop 7ebbe9d. I might started branch from wrong place.

BlackMorzan commented on src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/css/responsive.css:
```diff
@@ -1,3 +1,17 @@
+@media screen and (min-width: 2000px) {
+    .countdown_section {
+        padding-bottom: 323px;
+        padding-top: 320px;
+    }
+}
+
+@media screen and (min-width: 1200px) and (max-width: 1999px) {
+    .countdown_section {
+        padding-bottom: 150px;
+        padding-top: 150px;
+    }
+}
+
```
this chunk of code changes padding to make Santa visible on HD, fullHD, and 4K monitors - I think it is done for now (although 4K could use more changes to look good)

BlackMorzan commented:
I checked AdminServiceTest. It looks like it have been refactored and moved to IntegrationTests folder. I also added screenshots for 1024p, 1336p, and 1440p. I'd say it looks fine.

WTobor commented:
@BlackMorzan are we ready to complete this PR? Do you have Hactoberfest account ready? :)

BlackMorzan commented:
I'm ready! Let it by my 1'st Hactoberfest contribution :)
